app/services/concerns/xccdf/profiles.rb

```diff
@@ -30,6 +31,14 @@ def old_profiles
           ref_id: @op_profiles.map(&:id), benchmark: @benchmark&.id
         ).index_by(&:ref_id)
       end
+
+      def value_overrides(op_profile)
+        @cached_value_overrides ||= @op_value_definitions.index_by(&:id)
+        op_profile.refined_values.each_with_object({}) do |(value_id, selector), value_map|
+          value_definition = @cached_value_overrides[value_id]
+          value_map[value_id] = value_definition.value(selector)
```

Comment:
```suggestion
          value_map[value_id] = value_definition_for(ref_id: value_id).value(selector)
```

Reply:
I don't think this will work because `value_definition_for` returns a `ValueDefinition` but we need to call `.value(selector)` on the `OpenscapParser::Value`

Reply:
You can look into how is `op_source` used in rules :wink: